docs/docsite/rst/community/communication.rst

```diff
@@ -94,6 +94,14 @@ IRC meetings
 The Ansible community holds regular IRC meetings on various topics, and anyone who is interested is invited to
 participate. For more information about Ansible meetings, consult the `meeting schedule and agenda page <https://github.com/ansible/community/blob/master/meetings/README.md>`_.
 
+The Bullhorn
+------------
```

Comment:
This puts "The Bulhorn" as a subsection under "IRC Channels". It should be a separate section using the `====` markup